rules/transition_support.bzl

```diff
@@ -74,6 +113,21 @@ def _min_os_version_or_none(attr, attr_platforms, platform, attr_platform_type):
 
     return None
 
+def _bazel_7_objc_library_transition_values(settings, platform_type):
```

Comment:
Do you know if this can be done for Bazel 6 too? Thats the min. version we support now so if we can remove this conditionality that would be best

Reply:
apparently it works on bazel 6 as well.. check removed